_includes/footer.html

```diff
@@ -7,10 +7,9 @@
         open a <a href="https://github.com/techworkersco/techworkersco.github.io/issues">GitHub
         issue</a>.
         <br/>
-        ✊💛
+        ✊ 💛
         <br/>
         A <a href="https://techworkerscoalition.org">Tech Workers Coalition</a> project.
```

Comment:
should we just put this on the same line as the emoji now?

that might look better.

Reply:
Done.